### PR TITLE
Breadcrumb styling and tweak to pattern library structure

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1346362796)

### Description of Changes Made

- Adds styling for the breadcrumb, including making sure the forward slash is not read out by a screen-reader
- Moves the navigation components to a top-level folder to make them easier to locate in the pattern library
- Move the styleguide to be the first item in the pattern library so it is the first thing that loads
- Remove an unused sidebar tag that references an non-existent file
- Updates the styleguide
- Adds some reminders about the pattern library to the MR template.

### How to Test

With a local build running, visit either http://localhost:8000/pattern-library/render-pattern/patterns/navigation/components/breadcrumbs.html or any page in the site.

Ensure that at mobile only the immediate parent page shows.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-01-12 at 08 38 46
Screenshot 2024-01-12 at 08 38 36

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [ ] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] New font variants have not been added
- [ ] Not required

### CODE REVIEW

**Comment by helenb** [comment]
@lparsons396 I can talk you through the changes on this ticket.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -61,3 +61,9 @@
 - [ ] If JavaScript is needed alternatives have been considered
 - [ ] New font variants have not been added
 - [ ] Not required
+
+#### Pattern library
```
Comment (chris-lawton) [inline .github/pull_request_template.md:65]: Praise: Nice additions!

File: tbx/static_src/sass/components/navigation/_breadcrumbs-nav.scss [hunk 2]
```diff
@@ -1,46 +1,32 @@
...
 .breadcrumbs-nav {
+    @include font-size(supporting);
     display: flex;
-    align-items: center;
     flex-direction: row;
+    flex-wrap: wrap;
     width: 100%;
-    height: 100%;
-    margin-bottom: 10px;
+    margin: 15px 0 5px; // bottom margin of 5px is a 15px margin minus the 10px bottom margin on the li.
```
Comment (chris-lawton) [inline tbx/static_src/sass/components/navigation/_breadcrumbs-nav.scss:9]: Question: could this use `$spacer-mini` for 15px or is the spacing not part of the vertical spacing units?

File: tbx/static_src/sass/config/_mixins.scss [hunk 3]
```diff
@@ -140,6 +140,7 @@
     color: $color;
     text-decoration: underline;
     text-decoration-thickness: $link-underline-thickness;
+    text-underline-offset: 2px;
```
Comment (chris-lawton) [inline tbx/static_src/sass/config/_mixins.scss:143]: This looks better 👍

**Review by chris-lawton — APPROVED** [review]
Looks great, thanks @helenb.

Moving the styleguide to the top of the PL as well makes sense, nice to see a list components when you initially hit the library.

File: tbx/static_src/sass/components/navigation/_breadcrumbs-nav.scss (hunk 2 above)
Comment (helenb) [reply tbx/static_src/sass/components/navigation/_breadcrumbs-nav.scss]: good point - I've updated that. I've also removed some yaml variables that weren't being used.

**Pull request merged**